tasks/build-website-development.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
+
+gulp.task('build-website-development', ['prepare-website'], buildWebsiteEevelopmentTask);
+
+function buildWebsiteEevelopmentTask (cb) {
```

Comment:
Here and in other places: we don't need intermediate named function here, just pass it directly to `gulp.task`